src/conftest.py

```diff
@@ -122,6 +122,8 @@ def evaluate(self, example: Example) -> None | str:
         import_header.append(
             path_to_import_statement(Path(os.path.relpath(example.path)))
         )
+        # Add microgrid as default import
+        import_header.append("from frequenz.sdk import microgrid")
```

Comment:
I'm not sure if it is a good idea to add this, is just one extra line for examples and it makes examples more complete.